jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsSubsystemDefinition.java

```diff
@@ -87,14 +81,12 @@ public void registerAdditionalRuntimePackages(ManagementResourceRegistration res
                     RuntimePackageDependency.optional(RESTEASY_JSAPI.getName()),
                     RuntimePackageDependency.optional(RESTEASY_MULTIPART.getName()),
                     RuntimePackageDependency.optional(RESTEASY_YAML.getName()),
-                    RuntimePackageDependency.optional(JACKSON_CORE_ASL.getName()),
                     RuntimePackageDependency.optional(RESTEASY_CRYPTO.getName()),
                     RuntimePackageDependency.optional(JACKSON_DATATYPE_JDK8.getName()),
                     RuntimePackageDependency.optional(JACKSON_DATATYPE_JSR310.getName()),
                     // The following ones are optional dependencies located in org.jboss.as.jaxrs module.xml
                     // To be provisioned, they need to be explicitly added as optional packages.
                     RuntimePackageDependency.optional("org.jboss.resteasy.resteasy-jettison-provider"),
-                    RuntimePackageDependency.optional("org.jboss.resteasy.resteasy-jackson-provider"),
```

Comment:
Note that `org.jboss.resteasy.resteasy-jackson2-provider` is already present as it was added on line 79.